integrated_channels/sap_success_factors/exporters/course_metadata.py

```diff
@@ -0,0 +1,263 @@
+        return self.enterprise_configuration.provider_id
+
+    def transform_status(self, course_run):
+        """
+        Return the transformed version of the course status.
+        """
+        return (
+            self.STATUS_ACTIVE
+            if course_run['availability'] in [self.AVAILABILITY_CURRENT, self.AVAILABILITY_UPCOMING]
+            else self.STATUS_INACTIVE
+        )
+
+    def transform_title(self, course_run):
+        """
+        Return the transformed version of the course title, as well as the locale.
+        """
+        return [{
+            'locale': self.transform_language_code(course_run.get('content_language')),
+            'value': course_run.get('title') or ''
+        }]
+
+    def transform_description(self, course_run):
+        """
+        Return the transformed version of the course description, as well as the locale.
+        """
+        return [{
+            'locale': self.transform_language_code(course_run.get('content_language')),
+            'value': (
+                (course_run.get('full_description') or '')
+                or (course_run.get('short_description') or '')
+                or (course_run.get('title') or '')
+            )
+        }]
+
+    def transform_thumbnail_uri(self, course_run):
+        """
+        Return the transformed version of the course's thumbnail URI.
+        """
+        image = course_run.get('image') or {}
+        return image.get('src') or ''
+
+    def transform_content(self, course_run):
+        """
+        Return the transformed version of the course content.
+        """
+        return [{
+            'providerID': self.enterprise_configuration.provider_id,
+            'launchURL': self.get_launch_url(
+                course_run['enterprise_customer'],
+                course_run['key'],
+                course_run.get('enrollment_url') or ''
+            ),
+            'contentTitle': course_run.get('title') or '',
+            'contentID': course_run['key'],
+            'launchType': 3,
+            'mobileEnabled': course_run.get('mobile_available', 'false')
+        }]
+
+    def transform_price(self, course_run):  # pylint: disable=unused-argument
+        """
+        Return the transformed version of the course price.
+        """
+        return []
+
+    def transform_schedule(self, course_run):
+        """
+        Return the transformed version of the course schedule.
+        """
+        start = course_run.get('start') or UNIX_MIN_DATE_STRING
+        end = course_run.get('end') or UNIX_MAX_DATE_STRING
+        return [{
+            'startDate': parse_datetime_to_epoch_millis(start),
+            'endDate': parse_datetime_to_epoch_millis(end),
+            'active': current_time_is_in_interval(start, end)
+        }]
+
+    def transform_revision_number(self, course_run):  # pylint: disable=unused-argument
+        """
+        Return the transformed version of the course revision number.
+        """
+        return 1
```

Comment:
I don't actually know, but it used this value before, so I continue to pass this back to prevent a regression in however communication with SAPSF takes place.